cmd/update.go

```diff
@@ -60,40 +58,45 @@ func newUpdateCmd() *cobra.Command {
 func (uc *updateCmd) run() error {
 	flagVariablesMap = flagVariablesToMap(uc.flagVariables)
 
-	if uc.addon == "" {
-		addon, err := addons.PromptAddon(template.Addons, uc.provider)
-		if err != nil {
-			return err
-		}
-		uc.addon = addon
+	if dryRun {
+		dryRunRecorder = dryrunpkg.NewDryRunRecorder()
+		uc.templateVariableRecorder = dryRunRecorder
+		uc.templateWriter = dryRunRecorder
 	}
 
-	addonConfig, err := addons.GetAddonConfig(template.Addons, uc.provider, uc.addon)
```

Comment:
Are we not going to use user provided addon name and only support app routing for now?

Reply:
Yes, thats the only one we have and we can migrate this correctly to something better in the CLI in the future!